### ISSUE TITLE
Select element must have an accessible name

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1131]
on the home page:

```
Element Location:
select
<select role="listbox" label="language">
```

To solve this problem, you need to fix at least (1) of the following:

- [ ] Form element does not have an implicit (wrapped) <label>
- [ ] Form element does not have an explicit <label>
- [x] aria-label attribute does not exist or is empty
- [ ] aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- [ ] Element has no title attribute
- [ ] Element's default semantics were not overridden with role="none" or role="presentation"
